modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+= Uninstalling the CSI snapshot implementation
+
+To uninstall the installed CSI snapshot implementation, use the following procedure.
+
+.Prerequisites
+
+* {microshift-short} is installed and running.
+* You have deployed the CSI snapshot implementation.
```

Comment:
```suggestion
* The CSI snapshot implementation has been deployed on the MicroShift cluster.
```
Since it's deployed when there is no user input, it may be confusing to suggest the users performed the action.

Reply:
chnaged to "The CSI snapshot implementation is deployed on the MicroShift cluster"